crossing-training.lic

```diff
@@ -965,32 +963,10 @@ class CrossingTraining
     fput('get my wool yarn')
     dispose_trash('yarn')
   end
-
-  def forge(chapter, unique_name, item, trash_ingot)
-    unless @bought_ingot
-      buy_bronze_ingot
-      fput('stow my ingot')
-      @bought_ingot = true
-    end
-
-    return unless find_anvil
-
-    @bought_ingot = false
-
-    wait_for_script_to_complete('forge', ['trash', 'blacksmithing', chapter, unique_name, 'bronze', item])
-
-    # Handle the case in which forge fails (i.e. due to running out of oil)
-    pause 1
-    dispose_trash(checkright) if checkright.include? item
-    pause 1
-    dispose_trash(checkleft) if checkleft.include? item
-    pause 1
-
-    EquipmentManager.instance.empty_hands
-
-    return unless trash_ingot
-    fput('get my bronze ingot')
-    dispose_trash('ingot')
+  
+  def smith(full_name, noun)
+    wait_for_script_to_complete('smith', ['bronze', full_name, 'buy'])
+    dispose_trash(noun)
```

Comment:
I'm not sure if we ever call dispose trash while holding something else we care about.